tr55/makeMiniAppTable.py

```diff
@@ -44,9 +43,6 @@
 
     soil_types = ['a', 'b', 'c', 'd']
 
-    et_max = 0.207
```

Comment:
Yes, this particular code was not using the Pre-Columbian stuff, this was an unused variable.